action_files/test_models/src/data.py

```diff
@@ -35,9 +35,9 @@ def get_data():
     spec = [
         ['Country'],
         ['Country', 'State'], 
-        ['Country', 'Purpose'], 
+        # ['Country', 'Purpose'], 
```

Comment:
Why were these removed?

Reply:
Hierarchy needs to be strictly hierarchical

Reply:
I suggest we remove them instead of commenting them out.